extensions/ql-vscode/src/test-adapter.ts

```diff
@@ -220,14 +220,13 @@ export class QLTestAdapter extends DisposableObject implements TestAdapter {
       const state = event.pass
         ? 'passed'
         : event.messages?.length
-        ? 'errored'
-        : 'failed';
+          ? 'errored'
+          : 'failed';
       let message: string | undefined;
       if (event.diff?.length) {
         message = ['', `${state}: ${event.test}`, ...event.diff, ''].join('\n');
         testLogger.log(message);
       }
-      (event.diff || []).join('\n');
```

Comment:
Did we mean to use this above instead of `...event.diff`?

Reply:
No...in the previous line, it is testing that `event.diff` is truthy.

Reply:
And it's putting it into `message`